config/store.go

```diff
@@ -60,3 +61,47 @@ func NewStore(dsn string, watch bool) (Store, error) {
 
 	return NewFileStore(dsn, watch)
 }
+
+func MigrateConfig(sourceStore, destinationStore Store) error {
```

Comment:
Propose moving this into a `migrate.go` to leave `store.go` focussed on exposing the interface and constructor function.